methcomp/blandaltman.py

```diff
@@ -194,16 +170,7 @@ def blandaltman(method1, method2,
 
     """
 
-    plotter: _BlandAltman = _BlandAltman(method1, method2, x_label, y_label, title,
-                                         diff, limit_of_agreement, reference, CI,
-                                         xlim, ylim,
-                                         color_mean, color_loa, color_points,
-                                         point_kws)
-
-    # Draw the plot and return the Axes
-    if ax is None:
-        ax = plt.gca()
-
-    plotter.plot(ax)
+    ba: _BlandAltman = _BlandAltman(method1, method2, diff, limit_of_agreement, CI)
```

Comment:
If you don't want to return ax, you can make this a little cleaner by just doing
```
return _BlandAltman(method1, method2, diff, limit_of_agreement, CI)
```

Reply:
Yes, but with a public class now